src/archivematicaCommon/requirements/base.txt

```diff
@@ -6,16 +6,17 @@
 #
 agentarchives==0.6.0
 bagit==1.7.0
-certifi==2019.3.9         # via requests
+certifi==2020.4.5.1       # via requests
 chardet==3.0.4            # via requests
-django==1.8.19
-elasticsearch==6.3.1
+django==1.11.29
+elasticsearch==6.8.1
 idna==2.8                 # via requests
-mysqlclient==1.4.2.post1  # via agentarchives
+mysqlclient==1.4.6        # via agentarchives
```

Comment:
Since we're doing this, can we try to use the same version of mysql across al requirements?

Reply:
I synchronized all the common packages to their highest versions in https://github.com/artefactual/archivematica/pull/1603/commits/662873816e124a065915fea3a11ca2cef6c7b529

Reply:
Wait! The `mysqlclient` version is set indirectly by `agentarchives` not the `base.in` files!!?!?

Reply:
OK. We're using `mysqlclient==1.4.6` in all components now.